### DISCUSSION TITLE
Prefix not recognized in one of my RSS feed

### DISCUSSION THREAD

**Discussion by jmicaux** [skymethod/op3#37] [Q&A]
Hello There,

I have set up the OP3 prefix in 3 of my RSS feeds. Everything is OK for 2 feeds

https://www.3ds.com/contents/feed/podcast/global-operations-on-the-go
--> https://op3.dev/show/68fec42a-8e13-55b2-8d32-789cc6bff360

https://www.3ds.com/contents/feed/podcast/cloud-designing-impactful-innovation
--> https://op3.dev/show/e1bb2618-a2db-592f-9871-748ddf6e1b84

Unfortunately, I am stuck with this feed https://www.3ds.com/contents/feed/podcast/disruptors-unleashed

When I try to get the metrics, I always get the OP3 Prefix "not result" message

I guess something is not properly set in this RSS feed but I can't find the root cause

PS: Please note the feed is considered valid by Spotify, Apple, and YouTube Podcats

Any idea?

Regards

**Comment by jamescridland** [comment]
I can help - it's because Disruptors Unleashed isn't in the Podcast Index, which is required by OP3 to work.

I've just added it there for you.

This show doesn't also appear to be in Apple Podcasts either...?

**Reply by johnspurlock-skymethod** [reply]
Thanks James

That's correct, OP3 is intended for public feeds only, so it currently uses the podcast index as that signal.  Now that the feed for Disruptors Unleashed is present there, the corresponding OP3 page will be created in tonight's run

**Comment by jmicaux** [comment]
Hello James & John,

Awesome. Thanks for the feedbacks. 

Did I miss an action about the Podcast Index?
